### PR TITLE
Create "Edit Alarm" screen prototype

### PR DESCRIPTION
**Pull request by ChaoticWeg**
Title.

### CODE REVIEW

File: src/components/ToggleButton.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+interface ToggleButtonState {
+    isToggled: boolean;
+}
+
+const ANIMATION_TIME_MS = 75;
+
+export class ToggleButton extends React.Component<ToggleButtonProps, ToggleButtonState> {
+
+    public get isToggled(): boolean {
+        return this.props.isToggled;
```
Comment (richardkriesman) [inline src/components/ToggleButton.tsx]: Shouldn't this be
```suggestion
        return this.state.isToggled;
```
@richardkriesman? :p

**Review by richardkriesman — CHANGES_REQUESTED** [review]
Looks pretty good! If possible though, could we get some unit tests for `components/MainSettingsScreen/ScheduleListItem`? 

As a side note, integration tests for `screens/settings/EditAlarmScreen` would be nice, but it'd be better to write that next sprint when we're implementing models.

Click-through testing doesn't seem to reveal any issues!

**Comment by richardkriesman** [comment]
I'm more than willing to write tests for this if everyone else is busy!

File: src/components/ToggleButton.tsx (hunk 1 above)
Comment (richardkriesman) [reply src/components/ToggleButton.tsx]: Wait, this has apparently already been fixed. Cool

**Comment by ChaoticWeg** [comment]
> 
> 
> I'm more than willing to write tests for this if everyone else is busy!

yooooo

**Comment by richardkriesman** [comment]
[Prototype add alarm UI](https://trello.com/c/Ept1yioU/31-prototype-add-alarm-ui)

**Review by richardkriesman — APPROVED** [review]

**Pull request merged**